apps/meteor/tests/e2e/omnichannel/omnichannel-contact-center.spec.ts

```diff
@@ -262,4 +270,28 @@ test.describe('Omnichannel Contact Center', () => {
 			await expect(poContacts.findRowByName(EDIT_CONTACT.name)).toBeVisible();
 		});
 	});
+
+	test.only('Delete a contact', async () => {
+		await test.step('Find contact', async () => {
+			await poContacts.inputSearch.fill(DELETE_CONTACT.name);
+			await poContacts.findRowMenu(DELETE_CONTACT.name).click();
+			await poContacts.findMenuItem('Delete').click();
+		});
+
+		await test.step('Fill confirmation and delete contact', async () => {
+			await expect(poContacts.deleteContactModal).toBeVisible();
+			await expect(poContacts.btnDeleteContact).toBeDisabled();
+			await poContacts.inputDeleteContactConfirmation.fill('delete');
```

Comment:
Might be a good idea to alsp test what happens if the inserted value is incorrect. Will cover the case where any text enables the button.